libres_util/src/res_util_defaults.c

```diff
@@ -0,0 +1,24 @@
+/*
+   Copyright (C) 2011  Statoil ASA, Norway.
+
+   The file 'enkf_defaults.c' is part of ERT - Ensemble based Reservoir Tool.
+
+   ERT is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or
+   FITNESS FOR A PARTICULAR PURPOSE.
+
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+
+#include <res_util_defaults.h>
+
+/**
+   Currently no code here - but the way DEFAULT_STATIC_KW is implemented
```

Comment:
Currently no code here - and the historical argument with `DEFAULT_STATIC_KW` has long been irrelevant. Just remove the file completely.